### DISCUSSION TITLE
Restricting X-Forwarded-* header parsing

### DISCUSSION THREAD

**Discussion by Doctor-love** [quarkusio/quarkus#29887] [Q&A] [answered]
Hi y'all,

Is it possible in Quarkus to restrict from which IP addresses the X-Forwarded-*  headers are parsed/trusted?
A similar option exist in nginx called ["set_real_ip_from"](https://nginx.org/en/docs/http/ngx_http_realip_module.html#set_real_ip_from).  
  
My use-case is exposing a distributed application both via a load balancer and direct connections, but for traceability I don't want to enable trivial spoofing of source IP addresses. Something like "_quarkus.http.proxy.allow-forwarded-from=ip.address.of.loadbalancer_" would be useful.

**Comment by sberyozkin** [comment, marked as answer]
Hi @Doctor-love I've opened an enhancement request at #29888
